releasenotes/notes/fix-max-runtime-version-check-2a0c6d085efd1aac.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix an issue with the check of the maximum runtime version supported.
```

Comment:
Can we make this release note a little more user friendly. It would be nice to include the runtimes that will now be supported